docs/user/metacat/source/submitting.rst

```diff
@@ -25,6 +25,10 @@ Once you've registered and/or signed in, you will be returned to ``metacatui``.
 the upper right-hand corner and hover above your name, which will reveal a drop down menu. Click
 'Submit Data', and add your data and metadata documents & fill in the relevant and required fields.
 
+.. figure:: images/screenshots/image007_metacatuisubmitdata.png
+
+   To submit data, first sign in, then hover over your name, and then click 'Submit data'
```

Comment:
how do I "hover" and/or "click" on a tablet or phone? 😁 

How about something more agnostic, like:

"To submit data, first sign in, then choose your name to display a menu, and then select 'Submit data'"

(or whatever else makes sense in both contexts)?

Reply:
Thank you for the wording suggestion!